site/content/en/docs/tasks/dev/integrate_a_custom_job.md

```diff
@@ -93,22 +93,13 @@ Here are completed external integrations you can learn from:
 Add your framework's GroupVersionKind to `.integrations.externalFrameworks` in [controller_manager_config.yaml](
 https://kueue.sigs.k8s.io/docs/installation/#install-a-custom-configured-released-version).
 
-### RBAC Augmentation
-
-Kueue will need permission to get, list, and watch instances of your CRD.
-
-If you are building a custom Kueue deployment, simply add a `kubebuilder:rbac` annotation to a source code file
-(for example in [integrationmanager.go](https://github.com/kubernetes-sigs/kueue/blob/main/pkg/controller/jobframework/integrationmanager.go)) and regenerate the manifests.
-
-If you are deploying a Kueue release, modify either
-[charts/kueue/templates/rbac/role.yaml](https://github.com/kubernetes-sigs/kueue/blob/main/charts/kueue/templates/rbac/role.yaml)
-or [config/components/rbac/role.yaml](https://github.com/kubernetes-sigs/kueue/blob/main/config/components/rbac/role.yaml)
-to add the needed permissions.
-
 ### Job Framework
 
 Add a dependency on Kueue to your `go.mod`, import the `jobframework` and use it as described above to
-create your controller and webhook implementations. In the `main` function of your controller, instantiate the controller-runtime manager
+create your controller and webhook implementations. Note that in addition to adding RBAC Authorization for
+your CRD as described above, you also need to add RBAC Authorizations for the Kueue CRDs that will be
```

Comment:
it's not very clear what you mean by "above". Maybe you can reference the specific section.

Reply:
I reorganized and expanded a bit.  Hopefully it isn't too detailed now, but at least it should be unambiguous :)